src/request.ts

```diff
@@ -60,10 +60,11 @@ export class Request extends Readable {
     this.query = Object.keys(event.multiValueQueryStringParameters || {}).reduce(
       (queryParams, key) => {
         const value = event.multiValueQueryStringParameters![key] // cannot be null at this point
-        queryParams[key] = value[0]
+        const _key = key.replace(/\[\]/, '')
```

Comment:
As we are doing this for express compatibility, wouldn't it make sense to also parse `shoe[color]=white` into `{ shoe: { color: 'white' } }`?

Reply:
Yes, we can straight way use qs-parser as [link](https://github.com/expressjs/express/blob/master/lib/middleware/query.js#L27). But we are not having access to rawQueryString as we are using Payload format version 1.0.

Should we build our own logic ?

Reply:
we also have `?shoe[1]=white&shoe[0]=black` which will be converted to shoe : [black,white]. this can be done using qs-parser.

Reply:
it seems rather straightforward to construct the raw query string from `multiValueQueryStringParameters` and I'm also OK with switching to payload format version 2.0, I need to check what else changed.

Reply:
https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html

Reply:
yes, we can also construct it to string

Reply:
@sravan-archilogic I played around with it a bit, had to update `@types/aws-lambda` to latest which has some breaking changes.
It would be nice to support both version 1.0 and 2.0.

Reply:
I'll check the possibility.

Reply:
@mredele I have added support for both the version 1.0 and 2.0. can you have a look at it ?